projects/plugins/jetpack/modules/masterbar/admin-menu/admin-menu.css

```diff
@@ -207,10 +207,10 @@
 	background-color: #c3c4c7;
 }
 
-#adminmenu .toplevel_page_site-card:not(.has-site-icon) .wp-menu-image img[src^="data:image/svg"] {
-	height: auto;
-	padding-top: 7px;
+#adminmenu .toplevel_page_site-card:not(.has-site-icon) .wp-menu-image img {
```

Comment:
I'm not sure but I could see that updated selector apply to more menu item images of third party plugins

Reply:
Possibly. This would require that 3rd party plugins also name the menu `site-card` though, which seems unlikely. Famous last words of course.